components/style/animation.rs

```diff
@@ -1138,7 +1139,7 @@ impl ElementAnimationSet {
 
 #[derive(Clone, Debug, Eq, Hash, MallocSizeOf, PartialEq)]
 /// A key that is used to identify nodes in the `DocumentAnimationSet`.
-pub struct AnimationSetKey(pub OpaqueNode);
+pub struct AnimationSetKey(pub OpaqueNode, pub Option<PseudoElement>);
```

Comment:
Perhaps this should be a struct with named members instead now.

Reply:
Sure. This definitely improves some of the code in `script/animations.rs`.